.github/workflows/cla_bot.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     steps:
       - name: "CLA Assistant"
         if: (github.event.comment.body == 'recheck' || github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA') || github.event_name == 'pull_request_target'
-        uses: contributor-assistant/github-action/@master
+        uses: contributor-assistant/github-action@b3bbab0a75fa27270069933cec6f369c0b373b4e  # v2.2.1
```

Comment:
Just looking at their [guide](https://github.com/contributor-assistant/github-action/tree/v2.2.1) they've used `@v2.2.0`, is there any chance we could just use `@2.2.1`